custom_components/meural/pymeural.py

```diff
@@ -43,19 +51,31 @@ async def request(self, method, path, data=None) -> Dict:
             else:
                 kwargs["json"] = data
         with async_timeout.timeout(10):
-            resp = await self.session.request(
-                method,
-                url,
-                headers={
-                    "Authorization": f"Token {self.token}",
-                    "x-meural-api-version": "3",
-                },
-                raise_for_status=True,
-                **kwargs,
-            )
+            try:
+                resp = await self.session.request(
+                    method,
+                    url,
+                    headers={
+                        "Authorization": f"Token {self.token}",
+                        "x-meural-api-version": "3",
+                    },
+                    raise_for_status=True,
+                    **kwargs,
+                )
+            except ClientResponseError as err:
```

Comment:
this happens on token expiration.

Reply:
this exception is raised for anything that goes wrong making the request. Including other status codes. From https://github.com/GuySie/ha-meural/issues/28#issuecomment-688508468 it looks like only `err.status == 401` is an authentication exception and the others should just raise.